src/components/calculator/PersonRiskControls.tsx

```diff
@@ -88,6 +88,14 @@ export const PersonRiskControls: React.FunctionComponent<{
               }
             />
           </div>
+          {console.log(data.interaction)}
```

Comment:
Remove

Reply:
oops. can we add a lint rule about this so I get a slap on the wrist anytime I do it?!

Reply:
done.